adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -274,9 +275,9 @@ public void testFavorLocalCacheUseLocalRTFailsWithInvalidGrantSwitchToBroker()
      * will switch to broker.
      */
     @SmallTest
-    public void testFavorLocalCacheUseLocalRTSucceeds() throws PackageManager.NameNotFoundException,
-            NoSuchAlgorithmException, OperationCanceledException, IOException, AuthenticatorException,
-            InterruptedException {
+    public void testFavorLocalCacheUseLocalRTSucceeds()
+            throws PackageManager.NameNotFoundException, OperationCanceledException,
```

Comment:
I don't see any exceptions got changed in the throw part, and there is no runtime exception in the original throw. The only thing changed is the exception order and move the throw in the next line, that's why i'm asking. :)